installscripts/cookbooks/jenkins/recipes/configurejenkins.rb

```diff
@@ -141,15 +131,18 @@
   end
 end
 
-# TODO: we do this at the end, do we need it here?
-service 'jenkins' do
-  supports [:stop, :start, :restart]
-  action [:restart]
-end
+#Docker container has no service jenkins
```

Comment:
Can you verify if we need this second restart here by taking it out and seeing if everything still works?

Reply:
Verified for scenario1, yeah it is not needed.